lib/asciidoctor-bibtex/processor.rb

```diff
@@ -182,43 +200,47 @@ def build_bibliography_item(key, index = 0)
         puts "Failed to render #{key}: #{e}"
       end
 
+      result = []
       result << "[[#{key}]]" if @links
       if StyleUtils.is_numeric? @style
         result << "#{@bibtex_ob}#{index}#{@bibtex_cb} "
+        if RUBY_ENGINE == 'opal' and not cptext.nil?
+          # Numbering is added by asciidoctor-bibtex and therefore should be stripped here
+          cptext = cptext.first.gsub(/^[\[\d\.\]]*/, ""), ""
```

Comment:
You meant by Citation.js? Otherwise, I don't get why we do it only when the `RUBY_ENGINE == 'opal'`

Reply:
I meant asciidoctor-bibtex already adds numbering in the bibliography. But so does citation.js. So both add numbers. 

Actually the numbers from citation.js depend on the style. For instance for vancouver it is "1. MyBibEntry" whereas for ieee it is "[1] MyBibEntry". For asciidoctor-bibtex it is alwas [1].

To be consistent with asciidoctor-bibtex I strip the numbering from citation.js

Reply:
Understood.
I would say that "Strip Citation.js numbering, will be added by asciidoctor-bibtex"?

I think it's easier to understand if we explain what we are doing and then why. When I first read the sentence, I thought the goal was to remove the numbering added by asciidoctor-bibtex (and use Citation.js numbering but apparently that's the other way around).